CONTRIBUTING.md

```diff
@@ -4,4 +4,64 @@ Thank you for your interest in the Langium project! The following is a set of gu
 
 ## Prerequisites
 
-Developing Langium only requires `npm@^7.7.0`. For this you will need `Node.js 10` or greater.
\ No newline at end of file
+Developing Langium only requires `npm@^7.7.0`. For this you will need `Node.js 10` or greater.
+
+## Best practices for local development
```

Comment:
We should start the contributing guide with the instructions to develop the packages in our repository, and make that as simple as possible, to avoid creating the impression that we somehow need a complex setup. The explanations you added here are useful, but I'd put them into an "advanced" section to clarify that it's not always necessary.

Reply:
I agree. I was very verbose on a specific aspect that is maybe only relevant for 10% of future users.
The guide should be simple for the normal use-case which is straightforward and it should make this more obvious (re-structure, simple first, advance use case later).